src/seedu/addressbook/data/person/Address.java

```diff
@@ -8,12 +8,21 @@
  */
 public class Address {
 
+    public static final String DIVIDER = ", ";
     public static final String EXAMPLE = "123, some street";
-    public static final String MESSAGE_ADDRESS_CONSTRAINTS = "Person addresses can be in any format";
-    public static final String ADDRESS_VALIDATION_REGEX = ".+";
-
-    public final String value;
+    public static final String MESSAGE_ADDRESS_CONSTRAINTS = "Person addresses has to be in format BLOCK, STREET, UNIT, POSTAL_CODE";
+    public static final String ADDRESS_VALIDATION_REGEX = "^.+$";
```

Comment:
this one is too generic a regex; how do you enforce the format you are specifying now?